test/Dapr.Actors.AspNetCore.IntegrationTest/ActivationTests.cs

```diff
@@ -33,7 +33,7 @@ public class ActivationTests
         public async Task CanActivateActorWithDependencyInjection()
         {
             using var factory = new AppWebApplicationFactory();
-            var httpClient = factory.CreateClient();
+            var httpClient = factory.CreateClient(new Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactoryClientOptions { HandleCookies = false });
```

Comment:
Does this work on downlevel .NET versions? eg: is this API present pre-7.0?

Reply:
Yes, it is present for 3.1-7, though only 7 appears to handle cookies incorrectly. Given this comes from the integration suite, I believe it doesn't represent an issue with our standard code and I did manually test the actor example with net7 to ensure this isn't an issue.